resources/indexer.py

```diff
@@ -1,13 +1,227 @@
-#!/usr/bin/env python
+# indexer.py
+"""Data-indexing script for File Catalog."""
+
 import argparse
-from concurrent.futures import ProcessPoolExecutor
+import asyncio
 import hashlib
-import json
 import os
-import requests
-import sys
+import re
+import tarfile
+from concurrent.futures import ProcessPoolExecutor
+from datetime import date
 from time import sleep
 
+import requests
+import xmltodict
+import yaml
+from icecube import dataclasses, dataio
+from rest_tools.client import RestClient
+
+SEASONS = {
+    '2005': 'ICstring9',
```

Comment:
Not sure if the proper name for 2005 should be "IC1".  Not like it really matters, since no one uses that data anymore.

Reply:
I'm using what Kevin gave me, but I'm not reading data that early anyways.